indico/modules/events/contributions/templates/management/_subcontribution_list.html

```diff
@@ -46,17 +46,19 @@
                 <table class="i-table-widget tablesorter">
                     <thead>
                         <tr class="i-table">
-                            <th class="i-table checkbox-column"></th>
+                            <th class="i-table" data-sorter="false"></th>
+                            <th class="i-table checkbox-column" data-sorter="false"></th>
                             <th class="i-table id-column">
                                 {% trans %}ID{% endtrans %}
                             </th>
+                            <th class="i-table">{% trans %}Position{% endtrans %}</th>
```

Comment:
I think that column is more confusing than useful. The position is obvious from looking at the table